requirements/extras/sqs.txt

```diff
@@ -1,2 +1,3 @@
-boto>=2.8
+boto3
+botocore
```

Comment:
I think `boto3` already depends on `botocore`. Also, perhaps it would be a good idea to specify (at least) a minimum version?